client/js/analysis.js

```diff
@@ -62,8 +147,11 @@ function addDonutChart(str, percent) {
   div.className = 'item donut';
   div.style.setProperty('--percent', percent);
 
+  // Convert back to actual percentage
+  const percentString = String(100 - Number.parseInt(percent));
```

Comment:
`parseInt` is a global method, you don't need to do `Number.parseInt`

but also subtracting from 100 twice in the JS seems very janky compared to just subtracting from 100 in your CSS

Reply:
and converting it to a string just to parse it back to a number also seems janky, why not just pass the number into your method?

Reply:
The IDE tells me `.setProperty(string, string)` is the correct way. One way or another there needs be conversion. This is so small of a thing it doesn't matter.